kit/src/elements/input/mod.rs

```diff
@@ -78,10 +79,16 @@ pub fn validate_no_whitespace(val: &str) -> Option<ValidationError> {
     None
 }
 
-pub fn validate_alphanumeric(val: &str) -> Option<ValidationError> {
-    if !val.chars().all(char::is_alphanumeric) {
+// Default to requireing alpha-numeric inputs, unless ignore_colon override is set on the input field
+pub fn validate_alphanumeric(val: &str, ignore_colon: bool) -> Option<ValidationError> {
+    let mut val_no_colons = val.to_string();
```

Comment:
perhaps `val_no_colons` could be named `val` like it was before, because the value sometimes does contain colons